### PR TITLE
Cheatsheets: Multiple fixes for flagged warnings

### PR DESCRIPTION
**Pull request by altern8tif**
Changes made to cheatsheets to fix Travis CI tests.

Reference #2722.

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/github-flavoured-markdown.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
 	},
 	"aliases": [
 		"gfm",
-		"ghfmd"
+		"ghfmd",
+		"github flavored markdown",
+		"github markdown"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/github-flavoured-markdown.json]: Why have you added aliases? These won't fix any tests.

Comment (altern8tif) [inline share/goodie/cheat_sheets/json/github-flavoured-markdown.json]: Sorry, itchy fingers :grin: 

Shouldn't be on this PR, but just decided to put in the americanised-version of "flavoured" as an alias.

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/github-flavoured-markdown.json]: @altern8tif Lets remove the aliases - we should stick to the purpose of the PR (which is to fix section names).

Comment (altern8tif) [inline share/goodie/cheat_sheets/json/github-flavoured-markdown.json]: :+1: done.

**Comment by GuiltyDolphin** [comment]
@altern8tif Thanks, this looks good - I just want to check it out on Beta before we merge.

/cc @moollaza

**Comment by moollaza** [comment]
deploying to beta...

**Comment by GuiltyDolphin** [comment]
@altern8tif This has got some merge conflicts after a recent merge - if you `rebase` on top of that it should clear up.

**Comment by altern8tif** [comment]
@GuiltyDolphin check if it's all good?  I don't `rebase` much, so not too sure if everything's in its place...

**Comment by GuiltyDolphin** [comment]
@altern8tif Thanks! :dolphin:

**Pull request merged**